### PR TITLE
Initial stylelint setup

### PR DESCRIPTION
**Pull request by Henni**
validate our scss code in regard to code style

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,45 +1,47 @@
...
-		"karma-chai": "^0.1.0",
-		"karma-coverage": "^1.1.1",
-		"karma-firefox-launcher": "^1.0.0",
-		"karma-mocha": "^1.1.1",
-		"karma-mocha-reporter": "^2.1.0",
-		"karma-sinon": "^1.0.5",
-		"mocha": "^3.0.2",
-		"sinon": "^1.17.5"
-	}
+  "name": "contacts",
```
Comment (skjnldsv) [inline package.json]: Why is there spaces again? 😆

**Comment by skjnldsv** [comment]
Tests still pass if there is spaces instead of tabs

**Comment by Henni** [comment]
ignored some errors and fixed the remaining issues.

File: package.json (hunk 1 above)
Comment (Henni) [reply package.json]: thats what npm did by default...
I would leave it like that as it was automatically generated.

**Comment by Henni** [comment]
> Tests still pass if there is spaces instead of tabs

what do you mean by tests? travis?

**Comment by Henni** [comment]
ready for review
Please **squash and merge**

**Pull request merged**